notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -50,24 +50,6 @@
     "Note: Another interesting use case, not explored in this notebook, would be to test what is the accruacy when calibrating, removing the armband and giving it to someone else that is calibrating it on its own arm before testing it. The idea being that transfer learning might help getting better results (or faster calibration) than calibrating on one user."
```

Comment:
<p>NIT: Instead of comments, we try to write directly in the text cells of Jupyter: it is slightly more clean and more userfriendly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>